### PR TITLE
Build for Scala 2.11.7, 2.12.0-M1, 2.12.0-M2 and 2.12.0-M3

### PR DESCRIPTION
**Pull request by szeiger**
To be tagged as release 1.0.11.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,21 +4,23 @@ scalaModuleSettings
...
   else if (java.startsWith("1.8."))
-    Seq("2.12.0-M2")
+    Seq("2.12.0-M2", "2.12.0-M3")
   else
     sys.error(s"don't know what Scala versions to build on $java")
 }
 
-scalaXmlVersion            := "1.0.4"
+scalaXmlVersion            := {
+  if(scalaVersion.value == "2.12.0-M3") "1.0.5" else "1.0.4"
```
Comment (SethTisue) [inline build.sbt]: won't this break as soon as we do M4? how about `if(scalaVersion.value == "2.12.0-M2") "1.0.4" else "1.0.5"`

Comment (szeiger) [inline build.sbt]: When we do M4 (and it's not binary compatible), we need to update scala-partest anyway (and scala-xml may well be an even newer version then), so I don't think it really makes a difference which way you write that check.

**Comment by szeiger** [comment]
Do not release / tag this yet. Another bugfix is coming.

File: build.sbt (hunk 1 above)
Comment (szeiger) [inline build.sbt]: Actually, the way it is at the moment is easier. We need 1.0.5 for 2.12.0-M3 and 1.0.4 for 2.12.0-M2, 2.12.0-M1 and 2.11.

**Comment by szeiger** [comment]
Added the bugfix. From my POV this is ready to be released so I can use it from the Scala build.

**Comment by adriaanm** [comment]
LGTM

**Pull request merged**